0_graphing_functions.R

```diff
@@ -3346,7 +3346,7 @@ CarstenMetricChart <- function(plotnumber, chart_type, explicit_filename = "") {
     port <- bind_rows(combin, market)
 
     Type <- "Corporate bond"
-
+    # FIXME: Replace with dplyr::recode() or other options
     port$portfolio_name <- plyr::mapvalues(port$portfolio_name, c(portfolio_name_select, cb_market_ref), c("Portfolio", cb_market_ref))
 
     port$portfolio_name <- factor(port$portfolio_name, levels = c("Portfolio", cb_market_ref), ordered = TRUE)
```

Comment:
This is for another PR-- see  #184

Reply:
Anything in 0_graphing_functions can be ignored as this will be supersceded